src/Backend/Core/Engine/Model.php

```diff
@@ -707,7 +707,7 @@ public static function insertExtra(
                 'module' => $module,
                 'type' => $type,
                 'label' => $label ?? $module, // if label is empty, fallback to module
-                'action' => $action ?? $module, // if action is empty, fallback to module
+                'action' => $action ?? null,
```

Comment:
ModuleInstaller on master branch uses his own insertExtra method: https://github.com/forkcms/forkcms/blob/master/src/Backend/Core/Installer/ModuleInstaller.php#L500 - that's why it's not an issue on installation.